content/millennium/r4/proprietary-codes.md

```diff
@@ -78,6 +78,13 @@ The Appointment Resource supports proprietary codes for:
 
 * Appointment.serviceType codes are maintained in [Code Set 14249 Scheduling Appointment Type Synonyms](#code-set-14249-scheduling-appointment-type-synonyms)
 
+<h3>RelatedPerson</h3>
```

Comment:
needs to go under Individuals rather than Scheduling

Reply:
moved up https://github.com/cerner/fhir.cerner.com/pull/254/commits/a6cc265db68cabf4cb1d9abc1dec64f71d1c0b86#diff-f1a7812a3d6e9f4049fb6005eecfb1f5R72